jwst/ramp_fitting/tests/test_ramp_fit_cases.py

```diff
@@ -5,6 +5,18 @@
 
 from stdatamodels.jwst.datamodels import RampModel, GainModel, ReadnoiseModel, dqflags
 
+
+################## DEBUG ################## 
+#                  HELP!!
+import sys
+sys.path.insert(1, "/Users/kmacdonald/code/common")
```

Comment:
I don't think we want to have this included in the public code base. It's also causing CI and regtest errors.

Reply:
I removed this.